responses/responses.go

```diff
@@ -47,45 +52,48 @@ func BuildOkResponseWithData(dataName string, data interface{}) map[string]inter
 
 // SendResponse returns JSON response
 func SendResponse(w http.ResponseWriter, data map[string]interface{}) {
+	setDefaultContentType(w) // doesn't work if WriteHeader has been called on w first
 	json.NewEncoder(w).Encode(data)
 }
 
 // SendCreated returns response with status Created
 func SendCreated(w http.ResponseWriter, data map[string]interface{}) {
+	setDefaultContentType(w)
```

Comment:
I think this one is useless, because we already have it in `SendResponse`

Reply:
See the description. It's explained there.

Reply:
Oh ok, I see.